cogs/steam.py

```diff
@@ -0,0 +1,51 @@
+##############################
+# Created by: Brian <Jeewa#6972>
+##############################
+import aiohttp
+import discord
+import utils.log
+from discord.ext import commands
+
+
+class Steam(commands.Cog):
+    def __init__(self, bot):
+        self.bot = bot
+        self.log = utils.log.get()
+
+    # creating a library thats async
+    async def get_steam_featured(self):
+        async with aiohttp.ClientSession() as session:
+            async with session.get(
+                "https://store.steampowered.com/api/featuredcategories/"
+            ) as response:
+                steam = await response.json()
+                return steam["specials"]["items"]
+
+    # Create slash command for /steamsales
+    @commands.slash_command(
+        name="steamsales", description="View current sales on steam!"
+    )
+    async def Steam(self, ctx):
+        self.log.info(f"{ctx.author} said /steam.")
+        specials = await self.get_steam_featured()
+
+        sales_embed = discord.Embed(title="Today's Steam Sales:")
+
+        for game in specials:
+            sales_embed.add_field(
+                name=game["name"],
+                value=f"~~{game['original_price']}~~ {game['final_price']} ({game['discount_percent']}%)",
```

Comment:
Before you set the price in the embed, divide prices by 100 to get 00.00 pricing.

Reply:
Additionally, make sure you add a $ and USD to let people know these prices are US Dollars since there are a lot of Canadians

Reply:
hey made some changes ready to be reviewed!